### PR TITLE
chore: add scenario and diagram docs

### PR DESCRIPTION
**Pull request by NickLarsenNZ**
# Description

Initial scenario diagram for stackabletech/issues#524

- Page 1 is a scratch pad
- Page 2 has a more fleshed out scenario diagram
- Markdown doc covering the departments and personas. Needs more text to cover the actual scenarios, use cases, etc...

_Best edited via the drawio vscode plugin (to save having to export the image)._

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes
 
```[tasklist]
# Author
- [ ] Changes are OpenShift compatible
- [ ] CRD changes approved
- [ ] CRD documentation for all fields, following the [style guide](https://docs.stackable.tech/home/nightly/contributor/docs-style-guide).
- [ ] Helm chart can be installed and deployed operator works
- [ ] Integration tests passed (for non trivial changes)
- [ ] Changes need to be "offline" compatible
```

```[tasklist]
# Reviewer
- [x] Documentation added or updated. Follows the [style guide](https://docs.stackable.tech/home/nightly/contributor/docs-style-guide).
- [ ] Changelog updated
- [x] Cargo.toml only contains references to git tags (not specific commits or branches)
```

```[tasklist]
# Acceptance
- [ ] Feature Tracker has been updated
- [ ] Proper release label has been added
```

### CODE REVIEW

File: hack/world.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+Members of the team:
+
+- Pamela Scott (Team Lead)
+- Justin Martin
+- Isla Williams (contractor with specific user based permissions)
+
+## Marketing
+
+They need access
```
Comment (adwk67) [inline hack/world.md:38]: is the rest of the sentence missing here?

File: hack/world.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+Relevant members of the team:
+
+- Mark Ketting (needs read access to customer alytics dashboards)
```
Comment (adwk67) [inline hack/world.md:42]:
```suggestion
- Mark Ketting (needs read access to customer analytics dashboards)
```

**Comment by NickLarsenNZ** [comment]
>     * is there a file missing? The PR references 3 in all

There are supposed to be two files. The image has two pages (visible in drawio).
Now that I think about it, only one page will be used when displaying as an image, so I will split those (7d163a7).

>     * nit: I googled 2-3 of the fictional names and there are folk in the software industry with the same names. Probably difficult to get around. Maybe add a middle initial (which is not part of the username)?

Good point. Pretty unavoidable, but the middle initial is a good idea. Will do that (1eac592).

>     * would the CRA team access trino directly? I suppose they could with e.g. DbBeaver, or they could use the SQL sheets in Superset

If that makes sense as a scenario, sure. I have added an extra checkbox item to https://github.com/stackabletech/issues/issues/524 to write out some examples of what each user might be doing.

>     * nit: I guess the robot would typically write externally to a DMZ, but for our purposes it's fine to keep it simple. But: should there be some authorization layer there if it is writing directly to HDFS?

The Robot was added in haste to check off the "robot" requirement in https://github.com/stackabletech/issues/issues/524, but happy to flesh that out a little more, but I'm not quite sure what that might look like. IMO, a robot could even be a Spark job writing to HDFS. Twitter was more of a joke, but it could get getting data from an internal system (eg: telephony data).

Do you use vscode, and have the drawio plugin? Maybe it's easier for you to add a few boxes and arrows in this banch?

File: hack/world.md (hunk 1 above)
Comment (NickLarsenNZ) [reply hack/world.md:38]: Good catch. I ended up moving the access description to the actual persona.
I have revised the description in b7251ec

**Comment by NickLarsenNZ** [comment]
I spoke with @fhennig yesterday and he suggested not delving into groups yet (at least not for this ticket), but I still need to come up with various tasks that the personas would perform.

So freestyling here:

### Compliance and Regulation Analytics

> todo

### Customer Analytics

- **Isla**: Some of the telephony data is not available through APIs until an upgrade is complete. In the meantime, Isla has been brought in as a contractor to manually export production telephony data from the telephone system web UI. Isla runs the data through scripts (dropping PII data) which produce Parquet formatted file (call duration, time between call center staff calls, direct calls vs hunt-group calls). Isla then loads the files into the **Nonprod** HDFS under a specific path (`hdfs://customer-analytics/telephony/contact-center/*`) that she has access to. _This is not typical, but a necessary task_.

### Marketing

> todo

**Review by adwk67 — APPROVED** [review]
lgtm!

**Review by NickLarsenNZ — COMMENTED** [review]
Nice, thanks

**Pull request merged**